.gitattributes

```
@@ -0,0 +1 @@
+*.ipynb linguist-vendored
```

Nice, I don't know this feature ! May I suggest to add CSV and JSON as well ?

```suggestion
*.ipynb linguist-vendored
*.json linguist-vendored
*.csv linguist-vendored
```

---

Good one! Adding it